nncf/openvino/quantization/ignored_patterns.py

```diff
@@ -16,33 +16,63 @@
 OPENVINO_IGNORED_PATTERNS = Registry("IGNORED_PATTERNS")
 
 
-@OPENVINO_IGNORED_PATTERNS.register(IgnoredPatternNames.SOFTMAX_MATMUL)
-def softmax_matmul() -> GraphPattern:
-    pattern = GraphPattern()
+def _add_softmax_matmul(pattern: GraphPattern) -> None:
```

Comment:
You removed the following fused patterns for OpenVINO backend
- HWFusedPatternNames.STABLE_DIFFUSION
- HWFusedPatternNames.SOFTMAX_RESHAPE_MATMUL
- HWFusedPatternNames.SOFTMAX_RESHAPE_TRANSPOSE_MATMUL
- HWFusedPatternNames.SOFTMAX_RESHAPE_TRANSPOSE_GATHER_MATMUL
- HWFusedPatternNames.MATMUL_SOFTMAX_MATMUL

Did you verify that everything works correctly with new ignored patterns?

Reply:
I verified with the validation on the dedicated model scope. The accuracy numbers for them are equal. Also, I mannualy check the model graphs for GPT-2, swin, roberta models. They are equal.